email for course joiners.py

```diff
@@ -0,0 +1,46 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+# SMTP server details
+smtp_server = 'smtp.gmail.com'
```

Comment:
You don't need to add these, Mon.School is built on top of Frappe Framework which handles the email account on its own. To send a welcome email all you have to do is create a Notification using the Notification doctype when a new LMS Batch Membership gets created. You can [read this](https://docs.erpnext.com/docs/v13/user/manual/en/setting-up/notifications) to know more.